__init__.py

```diff
@@ -175,6 +182,10 @@ class Fast64Settings_Properties(bpy.types.PropertyGroup):
         name="Prefer RGBA Over CI",
         description="When enabled, fast64 will default colored textures's format to RGBA even if they fit CI requirements, with the exception of textures that would not fit into TMEM otherwise",
     )
+    point_lit: bpy.props.BoolProperty(
+        name="Supports Point Ligthing",
+        description="When enabled, fast64 will export expecting a gbi that supports point lighting, such as Majora's Mask's ucode",
+    )
```

Comment:
Just taking a quick look, this is not the right place for this setting, it should be with the microcode version.

Reply:
Good point sorry